scripts/yast2

```diff
@@ -295,8 +295,10 @@ if [ -n "$CMDLINE_HELP" ]; then
 fi
 ARGS=( )
 for i in "$@"; do
-  ARGS+=( "--arg" )
-  ARGS+=( "$i" )
+  if [ -n "$i" ] ; then
```

Comment:
as @mvidner already pointed out it should be fixed by different patch. This will explicitly filter all intentionally empty parameters, which is quite big change. so :-1: for me